packages/core/src/app/customer/__snapshots__/GuestForm.spec.tsx.snap

```diff
@@ -30,8 +30,6 @@ exports[`GuestForm matches snapshot 1`] = `
             >
               <label
                 class="form-label optimizedCheckout-form-label"
-                for="email"
```

Comment:
How come we're losing the `for/id`? Is this intentional?

Reply:
It was removed because I used a differnt way to create the label for `<EmailField/>` compoent.
All similar changes have been reverted.
All snapshots have been checked, no more removal like this.